pkg/common/constants.go

```diff
@@ -23,7 +23,8 @@ const (
 	ParameterKeyDiskEncryptionKmsKey = "disk-encryption-kms-key"
 
 	// Keys for Topology. This key will be shared amongst drivers from GCP
-	TopologyKeyZone = "topology.gke.io/zone"
+	TopologyKeyZone           = "topology.gke.io/zone"
+	KubernetesTopologyKeyZone = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
you don't want to translate this one?

Reply:
You're right, should be possible after this change: https://github.com/kubernetes/kubernetes/pull/77837